src/karmen_backend/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
This filters out some deprecation warning during tests in dependant lib (inlinecss), no need to worry about it.